.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,12 +26,20 @@
 
 ## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
 
+## Addeds tests?
+
+- [ ] yes
+- [ ] no, because they aren't needed
+- [ ] no, because I need help
+
 ## Added to documentation?
 
 - [ ] docs.dev.to
 - [ ] readme
 - [ ] no documentation needed
 
+## [optional] Post deployment tasks
```

Comment:
I'm not sure this is clear enough, should it be a question? For example, "Are there any post deployment tasks we need to perform?"

Your choice on this!

Reply:
Funny, I had a question first, then thought it's too long and shortened it. I take this to mean that my first gut feeling was correct and made it a question again, thanks!